components/HexagonIcons.js

```diff
@@ -7,41 +7,71 @@ import blockchain_dark from '../images/icons/hexagon-blockchain-dark.svg';
 import microchip_light from '../images/icons/hexagon-microchip-light.svg';
 import microchip_dark from '../images/icons/hexagon-microchip-dark.svg';
 
-export function RobotIconHexagon() {
+export function RobotIconHexagon(props) {
 	const theme = useTheme();
 	const logo = theme.mode == 'light' ? robot_light : robot_dark;
+	const {
+		width = 1080,
+		height = 935.31,
+		src = logo,
+		alt = 'Robot in a hexagaon with a Heart emoji face',
+		layout = 'responsive',
+		...other
+	} = props;
```

Comment:
I still don't think in es6 or whenever these patterns were introduced.
Decomposing props and spreading the remainder of props is clever as shit.